projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -473,6 +481,57 @@ public function invalidate_cache( $action = Filesystem_Utils::REBUILD_ALL ) {
 		return $this->invalidate_cache_for_url( home_url(), $action );
 	}
 
+	/**
+	 * Ignore certain cookies in the cache parameters so cached pages can be served to these visitors.
+	 *
+	 * @param array $parameters - The parameters with the cookies array to filter.
+	 * @return array - The parameters with cookies removed.
+	 */
+	public function ignore_cookies( $parameters ) {
+		static $params = false;
+
+		if ( $params ) {
+				return $params;
+		}
```

Comment:
I think a note about why this is necessary would help understand the reason its there.

Is this so it doesn't get executed multiple times?

Reply:
Yeah, it's to short circuit the main chunk of code in the function. In testing, I discovered it's only called once per request so this code isn't necessary right now. I may remove it.

Reply:
If a page isn't cacheable, then get_parameters and this code is called a few times so I'll leave it in, with a comment.